src/components/PostEditor/PostEditor.js

```diff
@@ -106,7 +106,8 @@ export default class PostEditor extends React.Component {
       toggleAnnouncementModal: false,
       showPostTypeMenu: false,
       titleLengthError: false,
-      dateError: false
+      dateError: false,
+      isDirty // prop tracking if form fields were touched from CreateModal
```

Comment:
it doesnt look like you are actually using this, why do you need to pass isDirty down to the PostEditor?